source/style/style.css

```diff
@@ -0,0 +1,581 @@
+  color: var(--black);
+  font-family: md;
+  font-size: 2rem;
+  transition: all 0.3s;
+}
+
+.nav-item .nav-link:hover {
+  color: var(--orange);
+  font-size: 1.1rem;
+  cursor: pointer;
+}
+
+.nav-item #link-active {
+  color: var(--orange);
+}
+
+/* Header Hero Section */
+.hero-wrapper {
+  background-image: linear-gradient(rgb(255, 255, 255), rgba(78, 56, 73, 0.73)), url('../images/main-bg.jpg');
+  background-position: center;
+  background-size: cover;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  padding: 20px 20px 50px;
+  margin-top: 1rem;
+}
+
+.hero-wrapper .greeting {
+  font-size: 1.6rem;
+  color: var(--orange);
+  font-family: sm;
+}
+
+.hero-wrapper .hero-header-text {
+  color: var(--orange);
+  font-size: 2rem;
+  font-family: lg;
+  margin-bottom: 30px;
+  text-align: left;
+}
+
+.hero-wrapper .hero-header-desc {
+  color: var(--gray);
+  font-size: 1.2rem;
+  border: 1px solid var(--gray);
+  padding: 0.6rem;
+}
+
+.hero-wrapper .hero-header-time {
+  margin-top: 1rem;
+}
+
+.hero-wrapper .hero-header-time,
+.hero-header-time .workout-time {
+  color: var(--gray);
+  font-size: 1.3rem;
+}
+
+.hero-header-time .workout-time {
+  margin-right: 0.2rem;
+}
+
+.workout-time i.fa-clock {
+  margin-right: 0.6rem;
+  font-size: 1rem;
+}
+
+/* Services Section */
+.services {
+  padding-right: 20px;
+  padding-left: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-image: url("../images/services.webp");
+  background-position: center();
+  background-size: cover;
+}
+
+.services .services-header {
+  margin: 1.5rem auto 2rem;
+  font-size: 1.8rem;
+  color: var(--gray);
+  padding-bottom: 1rem;
+  position: relative;
+  font-family: lg;
+}
+
+.services .services-header::after {
+  content: " ";
+  width: 20%;
+  display: block;
+  height: 1px;
+  position: absolute;
+  background-color: var(--orange);
+  margin: 1rem auto;
+  left: 40%;
+  right: 40%;
+}
+
+.card {
+  background-color: rgba(247, 230, 230, 0.1);
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  margin-bottom: 1rem;
+  border: 1px solid transparent;
+  transition: all 0.5s;
+}
+
+.card:hover {
+  border: 1px solid var(--gray);
+  transform: scale(1.1, 1.1);
+}
+
+.card .card-image i {
+  font-size: 3rem;
+  color: var(--gray);
+}
+
+.card-header {
+  color: var(--orange);
+  margin: 1rem 1rem;
+}
+
+.card-body p {
+  color: var(--gray);
+  font-family: sm;
+}
+
+.btn-join {
+  background-color: var(--orange);
+  padding: 1rem;
+  text-align: center;
+  text-decoration: none;
+  font-size: 1.2rem;
+  color: var(--gray);
+  margin-bottom: 3rem;
+  transition: all 0.3s;
+}
+
+.btn-join:hover {
+  background-color: #ac3f33;
+}
+
+/* Feature speaker */
+.feature-speaker {
+  background-image: none;
+}
+
+.change-height {
+  height: 250px;
+  overflow: hidden;
+  transition: all 0.5s;
+}
+
+.feature-card {
```

Comment:
- [ ] Kindly add a gap between the text and the image so it looks more like the given design. 😄 

![image](https://user-images.githubusercontent.com/92215878/190708609-13d05306-8e6d-4335-974b-283b7a1bdce4.png)